agency/README.md

```diff
@@ -143,7 +143,7 @@ Body Params:
 
 | Field        | Type    | Required/Optional | Field Description                                                    |
 | ------------ | ------- | ----------------- | -------------------------------------------------------------------- |
-| `vehicle_id` | String  | Required          | Vehicle Identification Number (vehicle_id) visible on vehicle               |
+| `vehicle_id` | String  | Required          | License Plate (if present) or VIN visible on a vehicle               |
```

Comment:
How will consumers of data know if it's plate or VIN? By validating it as a VIN and assuming it's a plate otherwise?

Reply:
Yes I think just based on the format of the string. This overloading of a field allows different cities and countries to use identifiers other than license plate, like VIN, or USDOT permit number, city permit, some custom placard, etc. Provides some flexibility instead of enumerating all the options in multiple fields.

Reply:
This is updated now to be an external identifier of sorts, with lots more vehicle info going into the new vehicle_attributes array.